banklib/src/lib.rs

```diff
@@ -0,0 +1,189 @@
+use std::io::{Read, Write};
+use std::net::TcpStream;
+
+use protocol_crate::{Command, Response, BankError, Operation};
+
+const SERVER_ADDRESS: &str = "127.0.0.1:7878";
+
+pub struct BankClient {
+    server_address: String,
+}
+
+impl BankClient {
+    pub fn new(x: &str) -> Self {
+        BankClient {
+            server_address: x.to_string(),
+        }
+    }
+
+    /// Creates a new account with the given `account` name.
+    ///
+    /// # Arguments
+    ///
+    /// * `account` - The name of the account to be created.
+    ///
+    /// # Returns
+    ///
+    /// * `Ok(usize)` - The ID of the newly created account.
+    /// * `Err(BankError)` - If the account already exists or there was an error during the process.
+    ///
+    pub fn create_account(&self, account: String) -> Result<usize, BankError> {
+        let response = self.send_command(Command::CreateAccount(account));
+        match response {
+            Response::Account(result) => Ok(result?),
+            _ => panic!("Unexpected create_account response: {:?}", response),
+        }
+    }
+
+    /// Increases the balance of the given `account` by the given `amount`.
+    ///
+    /// # Arguments
+    ///
+    /// * `account` - The name of the account to be increased.
+    /// * `amount` - The amount to be increased.
+    ///
+    /// # Returns
+    ///
+    /// * `Ok(usize)` - The ID of the newly created account.
+    /// * `Err(BankError)` - If the account already exists or there was an error during the process.
+    pub fn increase_account(&self, account: String, amount: u32) -> Result<(), BankError> {
+        let response = self.send_command(Command::IncreaseAccount(account, amount));
+        match response {
+            Response::OperationResult(Ok(_)) => Ok(()),
+            _ => panic!("Unexpected increase_account response: {:?}", response),
```

Comment:
Получаются, что если сервер вернет `Err`, мы тут спаникуем. Наверное, так не должно быть. Предлагаю вот так сделать:

```suggestion
            Response::OperationResult(result) => result,
            _ => panic!("Unexpected increase_account response: {:?}", response),
```